packages/components/src/components/codeExpression/codeExpression.tsx

```diff
@@ -0,0 +1,22 @@
+import { Component, h } from '@stencil/core'
+
+@Component({
+  tag: 'stencila-codeexpression',
+  styleUrl: 'codeExpression.css'
+})
+export class CodeExpression {
+  public static slots = {
+    text: 'text',
+    output: 'output'
+  }
+
```

Comment:
As above, should support `programmingLanguage` (but not prop, not slot?) and `errors` as slot.

Reply:
I intentionally omitted the `errors` functionality from this PR as I'd like to do some design work around UX, style, and interactivity.